### ISSUE TITLE
Support for Ubuntu 17.04

### ISSUE THREAD

**Issue by mikassi** [achaphiv/ppa-fonts#32]
Can you add support for Ubuntu 17.04?
I love you work. 

Thanks in advance.

**Comment by achaphiv** [comment]
Yeah, I've been trying to build it.

Something about zesty is different.

~~~
➤ docker run --rm -it ubuntu:xenial tty
/dev/console
➤ docker run --rm -it ubuntu:zesty tty
not a tty
~~~

**Comment by mikassi** [comment]
It seems to be working now. Thanks!

**Comment by detly** [comment]
Did this happen? There's no infinality package in the PPA for 17.04.

**Comment by achaphiv** [comment]
It's there.

https://launchpad.net/~no1wantdthisname/+archive/ubuntu/ppa/+packages?field.name_filter=&field.status_filter=published&field.series_filter=zesty

**Comment by detly** [comment]
I only see `cairo` and `freetype` packages, no `fontconfig-infinality` package. Is it still possible to tweak rendering parameters (like I did with the env vars in `/etc/profile.d/infinality-settings.sh` in Ubuntu 16.10)?

**Comment by achaphiv** [comment]
How did you get it in 16.10?

The `fontconfig-infinality` package is versioned 2013-01-04.  Which means it's been the same files copied for each new distribution. Since I saw no difference in 16.04 whether I used it or not, I just stopped packaging it altogether starting in 16.10.

Additionally, it was around that time I switched to Bohoomil's patches. They required a patch to fontconfig itself. I found it very painful trying to build it, and just gave up. I didn't really mind since the rendering was good enough for me.

I'm not an expert in debian packaging, so it's taken a lot of time and effort just to get this far. I can try again, but I'd have to see a clear evidence that it makes a difference.  Otherwise, it just doesn't seem to be worth the effort to maintain.

**Comment by detly** [comment]
Maybe I'm thinking of 16.04? I noticed a huge difference, possibly because
I use my monitors in portrait mode (even accounting for subpixel rendering,
it's hard to get good font rendering). Mostly the FIR coefficients, the
ChromeOS sharpening, and a bit of the stem alignment stuff.

I know a fair bit about Debian packaging, maybe I'll take a look this
weekend.

**Comment by detly** [comment]
I just checked my backups — it was 16.04.

**Comment by detly** [comment]
So I'm a bit confused — where's the actual upstream? [infinality.net](http://infinality.net) seems to be dead, and it's hard to identify the patches against the original Freetype. Where did the `fontconfig-infinality` package come from?

**Comment by achaphiv** [comment]
I manually made `fontconfig-infinality` through trial and error back in 2013. It's been so long that I don't actually remember how I did it. I vaguely recall downloading the files from infinality.net and then adding all the debian control files. It took many attempts before the [ackage was finally accepted by launchpad. This was before I made this repository to automate the package building. 

 It was extremely painful. In fact, if open the [fontconfig-infinality_20130104-0ubuntu0ppa1_all.deb](https://launchpad.net/~no1wantdthisname/+archive/ubuntu/ppa/+files/fontconfig-infinality_20130104-0ubuntu0ppa1_all.deb) and look at the `copyright` inside `data.tar.gz`, there's this lovely message:

```
Why is this so complicated? I don't care.

...
```
